packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -227,7 +222,6 @@ export default defineComponent({
     }
   },
   async created() {
-    await this.loadResourcesTask.perform()
```

Comment:
I think this needs to be called, no? Without, `LOAD_FILES` won't be called, hence the selection of spaces is not working.

Reply:
huh? can you elaborate? Seems to work for me, this was a wish by @kulmann

Reply:
The selection-checkbox works, but no space can be selected for me (sidebar, batch actions). Does this work for you?

<img width="752" alt="image" src="https://user-images.githubusercontent.com/50302941/223370379-4d819165-f500-4ca8-8d5a-fbaffc621012.png">

Reply:
reintroduced the usage of the loader task, but made the project space reloading optional (not used in mounted, only used in breadcrumbs click handler)